### PR TITLE
Move initialization of Brackets object from appshell_extensions.js to Global.js

### PR DESCRIPTION
**Pull request by nethip**
With newer versions of CEF, brackets object initialization inside `appshell_extensions.js` is causing a crash in the app. So moved the initialization to Brackets side where Brackets variable is defined. This was required as we are moving to a later version of CEF, 2785 on Linux only.

@abhijitapte @boopeshmahendran Could you please review this PR?

### CODE REVIEW

**Comment by nethip** [comment]
brackets-shell relevant PR: https://github.com/adobe/brackets-shell/pull/626

Note: Both the PRs need to be merged.

**Comment by boopeshmahendran** [comment]
@nethip The travis build is failing due to `ReferenceError: Can't find variable: appshell`. Is it expected?

**Comment by nethip** [comment]
@boopeshmahendran Not really. I will fix it.

File: src/utils/Global.js [hunk 1]
```diff
@@ -41,7 +43,14 @@ define(function (require, exports, module) {
     // inside Node for CI testing) we use this trick to get the global object.
     var Fn = Function, global = (new Fn("return this"))();
     if (!global.brackets) {
-        global.brackets = {};
+        // Earlier this initialization was happening inside appshell_extensions.js. But
+        // since the newer CEF versions have stronger JS checks render process was crashing
+        // siting JS eval error. So moved the initialization to here.
```
Comment (petetnt) [inline src/utils/Global.js]: nit: citing

Comment (nethip) [reply src/utils/Global.js]: @petetnt Would you want me to add the link to `appshell_extensions.js` file here?

Comment (petetnt) [reply src/utils/Global.js]: I think that would be useful, yes!

**Comment by nethip** [comment]
@petetnt I have added the file reference. Can you check if the PR is fine?

**Comment by navch** [comment]
LGTM 👍

**Review by abhijitapte — APPROVED** [review]
this looks good to me.

**Review by swmitra — APPROVED** [review]

**Pull request merged**